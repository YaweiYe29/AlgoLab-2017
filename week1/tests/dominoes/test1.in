6
6
2 3 1 2 1 1
5
3 1 1 1 1
100
7273 9129 2890 2859 8496 6120 7271 6661 2095 6642 6304 9264 8786 1474 8574 6464 9920 9925 2585 1211 385 4761 3861 4635 6561 2413 7906 7431 3997 9562 7224 5459 365 4494 4701 8441 9531 2290 9053 1961 3697 8697 8344 169 2893 1053 5451 115 6052 9687 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
100
22 28 9 9 26 19 22 20 7 20 19 28 27 5 26 20 30 30 8 4 2 15 12 14 20 8 24 23 12 29 22 17 2 14 15 26 29 7 28 6 12 27 26 1 9 4 17 1 19 30 1 1 2 2 2 1 1 2 2 1 2 1 2 2 2 2 2 1 2 2 2 2 2 2 2 1 1 1 1 2 1 1 2 2 1 2 1 2 1 1 2 2 1 1 1 1 1 2 2 1
100
14 30 14 19 8 26 24 23 19 23 19 9 13 8 18 1 13 26 21 29 7 28 30 23 18 30 10 2 1 21 14 19 24 27 9 14 16 20 12 30 22 28 9 16 4 13 29 8 24 13 1 1 2 1 1 2 2 2 2 1 2 1 1 2 1 2 1 1 1 2 2 1 1 2 2 2 2 2 1 2 1 1 1 2 1 1 1 2 2 1 2 2 1 1 2 1 2 2 2 2
1000
73 92 29 29 85 62 73 67 21 67 64 93 88 15 86 65 100 100 26 13 4 48 39 47 66 25 80 75 40 96 73 55 4 45 48 85 96 23 91 20 37 87 84 2 29 11 55 2 61 97 43 21 68 72 100 47 18 67 77 37 70 32 85 70 73 59 78 38 96 72 53 72 52 92 73 4 14 37 45 79 3 20 75 83 43 80 34 67 47 39 71 99 37 9 14 31 12 89 70 47 46 99 45 61 25 87 79 74 62 75 61 28 43 25 59 3 43 85 68 95 24 91 100 74 58 100 33 5 3 68 47 64 77 89 29 45 52 64 39 98 74 94 28 52 12 42 97 24 80 42 25 34 85 9 17 90 51 91 92 13 93 35 17 89 46 83 10 7 42 89 83 40 4 96 78 86 62 73 11 57 43 32 38 53 50 26 2 86 71 18 56 79 28 12 96 49 91 90 83 77 68 43 81 45 64 85 75 75 33 71 78 93 55 85 56 30 16 87 27 35 90 59 84 100 1 63 40 15 74 74 10 40 34 24 77 21 89 77 28 36 17 45 59 16 64 32 90 60 4 22 90 76 34 38 75 31 25 55 68 8 40 51 82 19 37 68 14 28 13 48 69 71 84 70 68 77 61 29 11 10 20 69 90 47 23 3 53 28 40 58 94 78 75 27 77 36 64 59 78 57 40 67 40 15 30 48 3 48 40 79 96 6 68 100 89 26 67 19 70 36 48 28 3 77 50 66 57 77 5 43 59 55 24 58 2 35 21 78 28 73 19 49 84 35 95 86 35 42 76 77 59 87 78 36 92 28 42 93 11 52 24 97 13 74 85 97 36 39 50 38 27 99 67 20 74 46 87 57 65 51 91 6 6 93 91 16 56 60 27 80 73 25 46 96 91 56 71 82 19 34 6 36 62 80 58 41 75 91 17 35 58 59 40 12 23 60 62 70 18 45 27 85 67 9 20 92 100 62 66 23 98 75 98 51 83 68 88 18 15 18 72 35 46 28 46 43 14 41 73 18 66 66 89 39 16 58 45 4 16 81 1 4 49 57 7 9 94 90 82 99 45 17 52 12 29 1 96 63 46 50 67 70 16 91 6 93 93 20 36 29 77 38 41 5 64 51 1 34 62 75 4 2 2 2 3 5 1 5 3 3 5 1 1 1 2 1 1 4 4 2 1 3 5 4 1 4 5 3 4 4 4 1 5 1 5 2 2 5 4 1 5 1 2 1 3 3 4 5 3 3 1 3 2 4 1 1 2 1 1 4 4 2 3 4 1 1 5 5 4 5 2 4 2 1 2 3 3 1 2 4 2 5 5 5 5 5 5 1 4 1 4 1 4 1 4 1 5 2 3 1 1 5 4 2 3 2 5 3 4 4 3 3 3 4 2 5 2 5 4 3 5 2 3 5 4 4 2 1 5 1 4 4 2 5 1 2 4 4 3 1 5 2 1 4 4 3 2 4 4 3 1 5 1 3 3 3 5 3 3 3 3 5 1 2 3 5 4 2 1 4 3 2 1 2 3 5 4 2 1 4 1 2 5 5 1 2 4 1 2 1 4 3 4 5 1 4 1 4 5 3 5 3 2 4 2 3 3 4 4 1 5 5 1 1 5 5 4 5 1 2 3 1 4 2 1 4 4 3 4 5 1 2 2 2 1 1 3 3 4 4 4 1 1 2 2 4 4 3 1 1 4 5 5 2 2 3 4 2 2 3 4 2 1 3 1 1 3 2 5 1 3 1 3 2 4 1 4 2 5 1 5 3 5 1 1 5 1 5 4 1 3 5 5 1 5 4 5 1 3 1 2 3 1 5 3 2 2 3 2 1 2 4 4 2 2 1 1 3 1 2 4 5 4 2 3 2 3 5 1 4 3 2 5 3 5 1 2 2 1 2 2 4 1 5 4 5 2 5 2 2 3 1 2 1 2 1 2 4 5 3 1 3 3 1 4 4 5 5 1 2 1 4 5 4 3 2 1 3 4 2 2 1 5 1 3 1 2 5 1 4 3 4 5 4 4 5 3 5 2 3 2 4 3 4 2 3 2 2 2 3 4 2 1 3 1 1 4 3 3 1 5 5 4 4 5 4 4 5 3 2 5 5 4 4 5 5 3 2 2 1 2 1 1 1 4 5 1 3 5 4 2 2 4 1 3 5 3 4 5 3 2 1 5 5 3 5 3 2 4 5 3 5 3 3 4 5 5 1 2 2 2 5 1 5 3 5 2 2 5 4 1 5 3 4 5 3 2 5 2 2
