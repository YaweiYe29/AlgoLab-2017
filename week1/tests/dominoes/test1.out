5
3
100
79
79
601
